File: adapter/src/main/java/com/hello7890/adapter/SpaceViewModule.java
package com.hello7890.adapter;

import android.view.ViewGroup;

import com.hello7890.adapter.vh.SpaceTViewHolder;

/**
 * 空module 占位
 */
public final class SpaceViewModule extends DataViewModule<Integer> {

    public SpaceViewModule(){
        this.dataList.add(0);
    }

    @Override
    public BaseViewHolder<Integer> onCreateViewHolder(ViewGroup parent, int viewType) {
        return new SpaceTViewHolder<>(parent);
    }
}

File: adapter/src/main/java/com/hello7890/adapter/SpaceViewModule.kt
package com.hello7890.adapter

import android.view.ViewGroup
import com.hello7890.adapter.vh.SpaceTViewHolder

/**
 * 空module 占位
 */
class SpaceViewModule : DataViewModule<Int>() {
    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): BaseViewHolder<Int> {
        return SpaceTViewHolder(parent)
    }

    init {
        dataList.add(0)
    }
}